tests/unittests/CoreGraphics.Drawing/CGPathDrawingTests.cpp

```diff
@@ -663,4 +663,61 @@ DRAW_TEST_F(CGPath, AddLinesWithoutMove, UIKitMimicTest<>) {
     CGContextStrokePath(context);
 
     CGPathRelease(thepath);
+}
+
+DRAW_TEST_F(CGPath, WidenTest_Stroking, UIKitMimicTest<>) {
+    CGContextRef context = GetDrawingContext();
+    applyBounds = GetDrawingBounds();
```

Comment:
It's static, and used in a bunch of tests here.  Just change this to local in each test.